src/App.tsx

```diff
@@ -1,7 +1,39 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: boolean,
+  keyBoardBtn : string,
+};
+
+export class App extends Component<{}, State> {
+  state: State = {
+    pressedKey: false,
+    keyBoardBtn: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ pressedKey: true, keyBoardBtn: event.key });
+    });
```

Comment:
You write here and inlines 21-22 almost the same code. When you would delete `pressesKey` variable. You could create a separate function for this purpose and use it in 2 places.